R/interface.R

```diff
@@ -194,6 +194,29 @@ dust_interface <- R6::R6Class(
     run = function(step_end) {
     },
 
+    ##' @description
+    ##' Set the "index" vector that is used to return a subset of data
```

Comment:
A default value of 0 to n-1 might be more intuitive default behaviour than returning an empty matrix

Reply:
yeah, I think you're right there